README.md

```diff
@@ -0,0 +1,39 @@
+# gitlab-jira-migration
+
+If you are using [JIRA](https://de.atlassian.com/software/jira) and [Gitlab](https://about.gitlab.com/) and you would like to change the url, user or password of your JIRA instance, you can not simply change the template and all the JIRA integration of all projects are changes (see [issue](https://gitlab.com/gitlab-org/gitlab-ce/issues/25541)).
+So you have to change every single project which is pretty anying. 
+
+Or you can use this script.
+
+## Dependencies
+
+You need [Requests](http://docs.python-requests.org/en/master/) for HTTP calls.
```

Comment:
Python Version? There is still a lot of Python 2 out there

Reply:
Should work both, but agree that this should be clarified.